docs/modules/ROOT/pages/backup-restore.adoc

```diff
@@ -0,0 +1,217 @@
+
+== Enabling Persistence
+--
+Before you can backup and restore Hazelcast data, you need to enable Persistence in the `Hazelcast` resource.
+
+. Create the `Hazelcast` resource:
++
+[source,yaml,subs="attributes+"]
+----
+include::ROOT:example$/hazelcast-persistence.yaml[]
+----
+<1> Base directory of the backup data.
+<2> <<recovery-policy, Cluster recovery policy>>.
+<3> Size of the PersistentVolumeClaim (PVC) where Hazelcast data is persisted.
++
+NOTE: Make sure to calculate the total disk space that you will use. The total used disk space may be larger than the size of in-memory data, depending on how many <<hot-backup, hot backups>> you take.
+
+. Apply the resource:
++
+[source,bash]
+----
+kubectl apply -f ./hazelcast.yaml
+----
+
+. Check that Hazelcast members are ready:
++
+[source,bash]
+----
+kubectl get pods -l app.kubernetes.io/instance=hazelcast
+----
++
+[source,bash]
+----
+NAME          READY   STATUS    RESTARTS   AGE
+hazelcast-0   1/1     Running   0          2m
+hazelcast-1   1/1     Running   0          1m
+hazelcast-2   1/1     Running   0          1m
+
+----
+
+. Check that Hazelcast PVCs are created:
++
+[source,bash]
+----
+kubectl get pvc -l app.kubernetes.io/instance=hazelcast
+----
++
+[source,bash]
+----
+NAME                                  STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+hot-restart-persistence-hazelcast-0   Bound    pvc-116b4084-a436-4462-b413-511b77df307b   20Gi       RWO            standard       2m
+hot-restart-persistence-hazelcast-1   Bound    pvc-a7711b6b-dcbf-45cb-9577-8ce6b1892f2f   20Gi       RWO            standard       1m
+hot-restart-persistence-hazelcast-2   Bound    pvc-64314d82-da7a-4e38-bd2d-e770a63dc4e8   20Gi       RWO            standard       1m
+----
+--
+
+[[hot-backup]]
+== Triggering a Single Hot Backup
+--
+. Create the `HotBackup` resource:
++
+[source,yaml,subs="attributes+"]
+----
+include::ROOT:example$/hot-backup.yaml[]
+----
+
+. Apply the resource:
++
+[source,bash]
+----
+kubectl apply -f ./hot-backup.yaml
+----
+
+--
+
+== Scheduling Hot Backups
+
+You can also take backups on schedule using the `spec.schedule` field.
+--
+. Create the HotBackup resource with `schedule` field filled:
++
+[source,yaml,subs="attributes+"]
+----
+include::ROOT:example$/hot-backup-scheduled.yaml[]
+----
++
+The field needs to take a valid cron expression. Some examples:
++
+[cols="1,1"]
+|===
+| ```30 10 * * *```
+| On 10:30 AM every day
+
+| ```0, 0, 1,15,25 * *```
+| On 1st, 15th and 25th each month, midnight
+
+| ```@monthly```
+| Once a month, first of the month, midnight
+|===
+
++
+For the full list of the supported expression, please check the link:https://pkg.go.dev/github.com/robfig/cron/v3@v3.0.1#hdr-CRON_Expression_Format[library documentation].
+
+. Apply the resource:
++
+[source,bash]
+----
+kubectl apply -f ./hot-backup-scheduled.yaml
+----
+--
+
+== Restoring from Hot Backups
+
```

Comment:
```suggestion
To restore a cluster from hot backups, you can reapply the `Hazelcast` resource, and it will have access to the PVCs that contain the persisted data.
```